honeybadgermpc/progs/triple_refinement.py

```diff
@@ -3,19 +3,19 @@
 import itertools
 
 
-def rename_and_unpack_inputs(field, a_, b_, c_, d, m):
+def rename_and_unpack_inputs(a_, b_, c_, d, m):
     n = m // 2  # number of triples
 
-    def pad(arr, k): return arr + [field(0)]*(d-len(arr))
+    def pad(arr, k): return arr + [0]*(d-len(arr))
     a, b, c = pad(a_[:n], d), pad(b_[:n], d), pad(c_[:n], d)
     x, y, z = pad(a_[n:], d), pad(b_[n:], d), pad(c_[n:], d)
 
     return a, b, c, x, y, z
 
 
 def get_extrapolated_values(poly, a, b, d, omega):
-    a_ = poly.interp_extrap(poly.interp_extrap(a, omega**2), omega)
-    b_ = poly.interp_extrap(poly.interp_extrap(b, omega**2), omega)
+    a_ = poly.interp_extrap_cpp(poly.interp_extrap_cpp(a, omega**2), omega)
+    b_ = poly.interp_extrap_cpp(poly.interp_extrap_cpp(b, omega**2), omega)
```

Comment:
why isn't interp_extrap once enough?

Reply:
I am just moving the code that I had used for benchmarking, this does not include the single interp_extrap like we discussed. I want to take that in a separate PR.

Reply:
This is anyway triple refinement, single `interp_extrap` will work only with random refinement.